app/components/primer/alpha/tool-tip-element.ts

```diff
@@ -176,18 +176,14 @@ class TooltipElement extends HTMLElement {
     return this.ownerDocument.getElementById(this.htmlFor)
   }
 
-  constructor() {
```

Comment:
We moved this into the `connectedCallback` to address:[no-constructor](https://github.com/github/eslint-plugin-custom-elements/blob/main/docs/rules/no-constructor.md)